src/utils/bindTooltip.js

```diff
@@ -30,14 +30,14 @@ export default function bindTooltip() {
           let posTop = getOffset(target).top - tooltip.offsetHeight - 10;
 
           if (posLeft < 0) {
-            posLeft = getOffset(target).left + target.offsetWidth / 2 - 20;
+            posLeft = (getOffset(target).left + target.offsetWidth) / (2 - 20);
             tooltip.classList.add('left');
           } else {
             tooltip.classList.remove('left');
           }
 
           if (posLeft + tooltip.offsetWidth > window.innerWidth) {
-            posLeft = getOffset(target).left - tooltip.offsetWidth + target.offsetWidth / 2 + 20;
+            posLeft = (getOffset(target).left - tooltip.offsetWidth + target.offsetWidth) / (2 + 20);
```

Comment:
@mmahalwy `posLeft = getOffset(target).left - tooltip.offsetWidth + (target.offsetWidth / 2) + 20;`
Sorry for late reply, didnt read the notifications!